docs/README.md

```diff
@@ -2510,28 +2510,15 @@ This extension mechanism is very new, but we're planning to port most of the fun
 - Composability: You can use the default rockets or configure your application to suit your needs without adding anything extra.
 - Easier to manage feature sets in different providers: It would be really hard for the core team and contributors to implement and test every new feature in every supported provider, so by providing functionality like rockets, you'll have access to the most advanced features for your provider faster, and the rockets library can be built on-demand for each provider.
 
-#### Naming recommendations
-
-There are no restrictions on how you name your rocket packages, but we propose the following naming convention to make it easier to find your extensions in the vast npm library and find related packages (code and infrastructure extensions cannot be distributed in the same package).
-
-- `rocket-{rocket-name}-{provider}`: A rocket that adds runtime functionality or init scripts. This code will be deployed along with your application code to the lambdas.
-- `rocket-{rocket-name}-{provider}-infrastructure`: A rocket that provides infrastructure extensions or implements deploy hooks. This code will only be used on developer's or CI/CD systems machines and won't be deployed to lambda with the rest of the application code.
+#### Create your own Rocket
 
-Notice that some functionalities, for instance an S3 uploader, might require both runtime and infrastructure extensions. In these cases, the convention is to use the same name `rocket-name` and add the suffix `-infrastructure` to the infrastructure rocket. It's recommended, but not required, to manage these dependent packages in a monorepo and ensure that the versions match on each release.
+> Currently, only available to extend your Booster infrastructure with AWS
 
-If you want to support the same functionality in several providers, it could be handy to also have a package named `rocket-{rocket-name}-{provider}-core` where you can have cross-provider code that you can use from all the provider-specific implementations. For instance, a file uploader rocket that supports both AWS and Azure could have an structure like this:
+To create a rocket that adds new functionality to your Booster app, you just need to create a npm package and add the characteristics your provider needs. For AWS, that's just a main class that contains two functions, `mountStack` and `unmountStack`. *Don't worry about them for now, we'll get to this shortly.*
```

Comment:
A rocket is nothing more than an npm package that extends your current Booster architecture.
The structure is simple and it mainly has 2 methods `mountStack` and `unmountStack`. We'll explain what they are in shortly.